seems something went wrong with `LPC55S69_S`, when running 
```
python tools/psa/release.py -m LPC55S69_S
```
throws
```

[2020-02-07T16:25:23.027Z] Compile [ 60.6%]: mbed_pinmap_default.cpp
[2020-02-07T16:25:23.027Z] [Error] mbed_pinmap_default.cpp@95,68: use of undeclared identifier 'serial_tx_pinmap'
[2020-02-07T16:25:23.027Z] [ERROR] ./hal/mbed_pinmap_default.cpp:95:68: error: use of undeclared identifier 'serial_tx_pinmap'
[2020-02-07T16:25:23.027Z]     static const int stdio_uart = pinmap_peripheral(STDIO_UART_TX, serial_tx_pinmap());
```
@mprse

--- a/components/testing/COMPONENT_FPGA_CI_TEST_SHIELD/test_utils.h
+++ b/components/testing/COMPONENT_FPGA_CI_TEST_SHIELD/test_utils.h
@@ -136,6 +136,7 @@ void find_ports(std::list<PortType> &matched_ports, std::list<PortType> &not_mat
                 }
             }
 
+#if DEVICE_SERIAL
             if (!strcmp(PortType::PinMap::name, UART_NAME) || !strcmp(PortType::PinMap::name, UARTNOFC_NAME)) {
                 if (pinmap_list_has_peripheral(pinmap_uart_restricted_peripherals(), port.peripheral)) {
                     utest_printf("Skipping %s peripheral %i with pin %s (%i)\r\n", pin_type,
@@ -143,6 +144,7 @@ void find_ports(std::list<PortType> &matched_ports, std::list<PortType> &not_mat
                     continue;
                 }
             }
+#endif
 
             // skipp pin searching if single pin port type
             if (PortType::pin_count > 1) {

--- a/hal/mbed_pinmap_default.cpp
+++ b/hal/mbed_pinmap_default.cpp
@@ -90,6 +90,7 @@ MBED_WEAK const PinList *pinmap_gpio_restricted_pins()
 }
 
 //*** Default restricted peripherals ***
+#if DEVICE_SERIAL
 MBED_WEAK const PeripheralList *pinmap_uart_restricted_peripherals()
 {
     static const int stdio_uart = pinmap_peripheral(STDIO_UART_TX, serial_tx_pinmap());
@@ -104,3 +105,4 @@ MBED_WEAK const PeripheralList *pinmap_uart_restricted_peripherals()
     };
     return &peripheral_list;
 }
+#endif

--- a/hal/pinmap.h
+++ b/hal/pinmap.h
@@ -174,7 +174,9 @@ const PinList *pinmap_restricted_pins(void);
  *
  * @return Pointer to a peripheral list of peripheral to avoid
  */
+#if DEVICE_SERIAL
 const PeripheralList *pinmap_uart_restricted_peripherals(void);
+#endif
 
 /**
  * Get the pin list of pins to avoid during GPIO/GPIO_IRQ testing

